Lombiq.TrainingDemo/Lombiq.TrainingDemo.csproj

```diff
@@ -48,11 +42,15 @@
     <PackageReference Include="OrchardCore.DisplayManagement" Version="1.8.2" />
   </ItemGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.OrchardCore\Lombiq.HelpfulLibraries.OrchardCore.csproj" />
-    <ProjectReference Include="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.csproj" />
+  <!-- The two Helpful Libraries inclusions are necessary to support using the module both from its standalone solution,
+  as well as from https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions. -->
```

Comment:
What does "two" refer to here? If you meant including `Lombiq.HelpfulLibraries.OrchardCore` two ways, maybe this would be clearer.
```suggestion
  <!-- This module requires Lombiq.HelpfulLibraries.OrchardCore, and it can be included two ways. The PackageReference
  is used by the standalone solution, while ProjectReference is needed when including the module in
  https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions. Only one of the two is ever active. -->
```

Reply:
Yep, meant that, thanks.